CHANGELOG.md

```diff
@@ -14,28 +14,104 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+-   `isNewPrimaryUser` is renamed to `createdNewUser`
+-   Refactored/renamed some styling options (`resetPasswordHeaderTitle` -> `headerTitle withBackButton`)
+-   Some default styling has changed related to how fonts/font-weights are applied
+-   Changed how `headerSubtitle` is styled in components: `EmailPasswordResetPasswordEmail`, `EmailPasswordSubmitNewPassword`, `EmailPasswordSignInHeader`, `EmailPasswordSignUpHeader`
+
+#### Migration guide
+
+##### Success redirections & isNewPrimaryUser rename
+
+Before:
+
+```tsx
+import SuperTokens, { SuperTokensWrapper, getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
+import Passwordless from "supertokens-auth-react/recipe/passwordless";
+
+SuperTokens.init({
+    appInfo: {
+        // appInfo
+    },
+    recipeList: [
+        Passwordless.init({
+            contactMethod: "EMAIL_OR_PHONE",
+
+            async getRedirectionURL(context) {
+                if (context.action === "SUCCESS") {
+                    if (context.isNewRecipeUser) {
+                        // new primary user
+                    } else {
+                        // only a recipe user was created
+                    }
+                }
+                return undefined;
+            },
+        }),
+        Session.init(),
+    ],
+});
+```
+
+After:
+
+```tsx
+import SuperTokens, { SuperTokensWrapper, getSuperTokensRoutesForReactRouterDom } from "supertokens-auth-react";
+import Passwordless from "supertokens-auth-react/recipe/passwordless";
+
+SuperTokens.init({
+    appInfo: {
+        // appInfo
+    },
+    async getRedirectionURL(context, userContext) {
+        if (context.action === "SUCCESS") {
+            if (context.createdNewUser && context.recipeId === "passwordless") {
+                // custom logic
+            }
+        }
+        // Returning undefined means we use the default redirection
+        return undefined;
+    },
+    recipeList: [
+        Passwordless.init({
+            contactMethod: "EMAIL_OR_PHONE",
+        }),
+        Session.init(),
+    ],
+});
+```
+
+#### Styling changes
+
+There are 2 types of style changes in this release:
+
+1.  Refactored/renamed some styling options. If you didn't add custom styles for the following classes there is no action needed
+
+    -   `resetPasswordHeaderTitle`: should now be styled using `withBackButton`
+    -   `headerSubtitle`: now directly includes the text instead of a `div`, your styles may require an update to match this change
+
+2.  Some default styling has changed related to how fonts/font-weights are applied. This may interact with your own font/sizing settings,
+    if you didn't add font, font-size/weight and size changes to your custom styles, no action should be needed.
+
+Both of the above changes could have subtle interactions with custom styles, so we recommend a quick manual (visual) test to make sure everything
+still looks right.
 
 ### Changes
```

Comment:
In changes, we should mention that there are new support codes specific to MFA and account linking.